src/smexperiments/tracker.py

```diff
@@ -591,10 +581,7 @@ def log_confusion_matrix(
         output_artifact=True,
     ):
         """Log a confusion matrix artifact which will be displayed in
-        studio.  Requires sklearn.
-
-        Note that this method must be run from a SageMaker context such as studio or training job
```

Comment:
but those methods indeed can only be called within sagemaker env?

Reply:
`CreateTrialComponent` and `CreateArtifact` limitations were recently lifted.

Reply:
got it